operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainResource.java

```diff
@@ -996,6 +1000,28 @@ private void addFatalValidationFailures() {
       whenAuxiliaryImagesDefinedVerifyOnlyOneImageSetsSourceWDTInstallHome();
       whenDomainCreationImagesDefinedVerifyOnlyOneImageSetsSourceWDTInstallHome();
       verifyIntrospectorEnvVariables();
+      verifyModelMotConfiguredWithInitializeDomainOnPV();
```

Comment:
Looks like a typo in method name ->
verifyModelMot -> verifyModelNot

Reply:
good catch. fixed.